src/types/props/home/index.ts

```diff
@@ -3,4 +3,5 @@ import { IPost } from '@/types/post'
 export interface PostsProps {
   posts: IPost[],
   toggleEditPost: (postId: string) => void
+  toggleSetPrivacyModal: () => void
```

Comment:
cái này có tác dụng gì suốt toàn bộ các component thế? sao cứ truyền qua truyền lại với cái togglePrivacyModal làm gì